projects.md

```diff
@@ -27,60 +31,67 @@ The functional specification details:
 For tool projects, the users are typically programmers, and so the functional specification describes the programming interface.
 
 ### Steps 4 and beyond: Iteratively Develop And Refine the Project
+
 You will organize the project as a set of short-term deliverables. Typically, you focus first on those parts where you have the most uncertainty since projects typically fail because of "unknown unknowns". For some projects, this will be a data-first focus to make sure that your data can answer the questions that you pose. For others, it may be exploring a python package that you hope will provide key features (e.g., visualizations).
 
 ## Project Structure
-Projects should have a online github repository with the project name. Top level folders/files within the repository include:
+
+Projects should have an online GitHub repository with the project name. Top level folders/files within the repository include:
 
 * README.md file that gives an overview of the project
 * LICENSE file
 * setup.py file that initializes the project after it has been cloned
 * doc folder that contains documentation (including the functional specification, the design specification, and the final project presentation or poster)
-* python package folder (with the same name as the repository) that is structured as one or more python modules (e.g., with __init__.py files) and test files that begin with "test_".
+* python package folder (with the same name as the repository) that is structured as one or more python modules (e.g., with **init**.py files) and test files that begin with "test_".
```

Comment:
I think this needs to keep the underscores but use backticks to make them render properly.